dropwizard-oauth/src/test/groovy/smartthings/dw/oauth/SpringSecurityAuthenticatorSpec.groovy

```diff
@@ -273,4 +275,26 @@ class SpringSecurityAuthenticatorSpec extends Specification {
         actual.get().additionalInformation == ["principal": "device:1234", "exp": 3025890541]
     }
 
+    def 'token with trailing authorization headers are cleaned correctly'() {
```

Comment:
Wouldn't you need an `@Unroll` on this method in order to leverage the `where` block?

Reply:
No, the @Unroll will only make each row in the where-block be treated as a separate test case. Now all the rows will be treated as a single test case.